Add docs page covering catalog pricing details (#2520)
chainguard-dev/edu: content/chainguard/chainguard-images/pricing.md

> ---

Reviewer: I don't think pricing should be at the top level of Chainguard Containers. "About" makes sense to me, but open to discussing other options.

Author: Fair. The page is really about what customers get with what we call "catalog pricing" and not about actual pricing. What if it was called "About Catalog Pricing"?

Reviewer: The file title is fine, but you can change it to `catalog-pricing.md` if that makes more sense, I am talking about what directory it lives in

Author: Got it. Moving the page with the next commit.

title: "Chainguard Container Catalog Pricing"
linktitle: "Catalog Pricing"
type: "article"
description: "Explains Chainguard Container catalog pricing, repository quotas, renaming, and sync, and helps you understand how to make the most of your subscription."
date: 2025-08-19T08:49:31+00:00
lastmod: 2025-08-19T08:49:31+00:00
draft: false

> tags: ["Chainguard Containers", "pricing"]

Reviewer: I would keep to [approved tags](https://github.com/chainguard-dev/edu/blob/main/TAG_GUIDELINES.md) vs. adding a random one-off here

images: []
menu:
  docs:
    parent: "chainguard-images"
weight: 055
toc: true
---

> Catalog pricing is Chainguard's pricing model for container usage that replaces the manual ticket process for adding images to an organization. This enables you to self-service request individual images in the console to be added to your repository from the wider Chainguard catalog.

Reviewer: Since this is documentation and not an announcement, I don't think we need to say it replaces another process.

Author: I thought that was part of why end users would care. Streamlining the process by removing a typical step (that still exists in our classic pricing model) is part of the value proposition.

If you still think it needs to go, I'll remove it.

> Catalog pricing is Chainguard's pricing model for container usage that replaces the manual ticket process for adding images to an organization. This enables you to self-service request individual images in the console to be added to your repository from the wider Chainguard catalog.

Reviewer: ```suggestion
Catalog pricing is Chainguard's pricing model for container usage that replaces the manual ticket process for adding images to an organization. This enables you to request individual images in the console to be added to your repository from the wider Chainguard catalog.
```

> The catalog pricing model is an option that requires you to be on the Catalog Pricing plan.

Reviewer: Should "Catalog Pricing" be title case? If so, it should be consistent throughout (first sentence it's not title case, for instance).

> > NOTE: If you would like to provide feedback, need troubleshooting assistance, or if your organization is not yet participating and would like to, please reach out to your account team.

Reviewer: Is there a rationale for the note on this page?

Author: It is similar to a note I read on the [private apk repo page](https://edu.chainguard.dev/chainguard/chainguard-images/features/private-apk-repos/) (near the bottom under Troubleshooting).

I was thinking that a little guidance might be useful since the only other instruction on how to sign up is in a draft of some customer comms. Maybe the note is too obvious.

## Catalog Pricing

> Chainguard offers catalog pricing for our container catalog, providing access across the Chainguard Container images catalog.

Reviewer: I would make this the first sentence of this doc (replacing the one there now). It's a bit redundant in this section.

- **What’s included**: Unlimited access to the full Chainguard-maintained container image catalog.
- **Benefits**:  

>   - One subscription covers the entire catalog if your plan includes FIPS, otherwise it includes all non-FIPS catalog options.

Reviewer: Are AI images included?

Author: I'll have to ask. They weren't mentioned separately in conversations or materials I found.

  - No more per-repository licenses.
  - Predictable monthly or annual costs.

> - **Use cases**: Perfect for organizations that pull many different Chainguard images across multiple teams or projects.

Reviewer: ```suggestion
- **Use cases**: Ideal for organizations that pull many different Chainguard images across multiple teams or projects.
```

## Add a Container Image

To add an image to your organization, select **Images** in the sidebar menu and then select the **Chainguard catalog** tab.

> Scroll through the list or use search to find the image you want. If the images is available to your organization, use the **Add to org** button in the row for the image listing. If your organization doesn't not have access to FIPS images, you will not be able to add them. The **Add image** dialog will appear. Click **Add image** to close the dialog and finish.

Reviewer: ```suggestion
Scroll through the list or use search to find the image you want. If the image is available to your organization, use the **Add to org** button in the row for the image listing. If your organization doesn't not have access to FIPS images, you will not be able to add them. The **Add image** dialog will appear. Click **Add image** to close the dialog and finish.
```

> > NOTE: If you see **Available in org** in the place of the add button, return to **Images** and click the **Add image** button above the list of organization images. Use search in the dialog that appears to find the image and click the bookmark icon next to the image name. You will be required to enter a different name for the new image, which will be added alongside the existing image when you click **Add image**.

Reviewer: If it is available to the org, doesn't it mean the user can pull down the image?

Author: Not in this context, it means the image already exists in the org so there isn't a need to add it for the org to access unless some customization or duplication of the image is needed.

> It can take up to 60 minutes for the image to be added to your organization, but often happens in as few as 10 minutes.

Reviewer: ```suggestion
Please allow up to 60 minutes for the image to be added to your organization. 
```

## Repository Quotas

> Your subscription includes a repository quota, which is the number of private repositories you can create or link. **custom assembled images you can add**

Reviewer: What is: 

```
**custom assembled images you can add**
```

> - **Checking quota**: Go to **Settings → Plan & Subscription** to see your current quota and usage.

Reviewer: I would make this prose instead of a bulleted list, it's not consistent with the rest of this document.

- **Exceeding quota**:
  - You’ll receive a warning if you approach your limit.
  - To continue adding repositories, upgrade your plan or remove unused ones.
- **Public repositories**: May not count toward your quota (check your plan details).

## Repository Sync

The repository sync feature keeps your repositories aligned with the upstream Chainguard catalog.  

> - **What it does**:  

Reviewer: The bullets here also don't feel consistent.

  - Automatically updates your repository with the latest secure image builds.  
  - Helps ensure compliance and reduces supply-chain risks.  
- **When to use**: Ideal for teams that want guaranteed up-to-date images without manual intervention.  

## Learn More

> 

Reviewer: It would be nice to add an intro sentence here.

> - [Blog announcement: Unlock the Full Chainguard Containers Catalog – Now with a Catalog Pricing Option](https://www.chainguard.dev/unchained/unlock-the-full-chainguard-containers-catalog-now-with-a-catalog-pricing-option)

Reviewer: I would add utm in both links

description: "Explains Chainguard Container catalog pricing, repository quotas, renaming, and sync, and helps you understand how to make the most of your subscription."
date: 2025-08-19T08:49:31+00:00
lastmod: 2025-08-19T08:49:31+00:00
draft: false
tags: ["Chainguard Containers", "pricing"]
images: []
menu:
  docs:
    parent: "chainguard-images"
weight: 055
toc: true
---

Catalog pricing is Chainguard's pricing model for container usage that replaces the manual ticket process for adding images to an organization. This enables you to self-service request individual images to be added to your repository from the wider Chainguard catalog.

The catalog pricing model is an option that requires you to be on the Catalog Pricing plan.

> NOTE: If you would like to provide feedback, need troubleshooting assistance, or if your organization is not yet participating and would like to, please reach out to your account team.

---

## Catalog Pricing

Chainguard offers catalog pricing for our container catalog, providing access across the Chainguard Container images catalog.

- **What’s included**: Unlimited access to the full Chainguard-maintained container image catalog.
- **Benefits**:  

>   - One subscription covers the entire catalog.  

Reviewer: What about AI images?
